data/mods/Xedra_Evolved/overmap/mutable_special.json

```
@@ -7,7 +7,7 @@
     "city_distance": [ 1, 40 ],
     "occurrences": [ 80, 100 ],
     "flags": [ "UNIQUE", "WILDERNESS" ],
```

```suggestion
    "flags": [ "OVERMAP_UNIQUE", "WILDERNESS" ],
```
Not sure why it didn't yell that UNIQUE isn't valid anymore but it's currently trying to spawn 80 to 100 times not 80% of the time c;

---

All eruptions all the time!